static/css/newcommunity.css

```
@@ -0,0 +1,777 @@
+.SandboxRoot.env-bp-430 .timeline-Tweet-text {
+
+  font-size:13pt !important;
+}
+
+.twittercol1 {
+  width: 60%;
+  margin-left:20%;
+  float:left;
+  padding:2%
+}
+.fa {
+  font-size: 25px;
+  color: #FFFFFF;
+}
+.panel-title {
+ font-size: 25px;
+ color: #FFFFFF;
+ font-family: 'Open Sans', sans-serif;
+}
+.panel-heading {
+ background-color: #1da1f2 !important;;
+ color: #1da1f2 !important;;
+}
+.panel {
+ margin-top: 25px;
+}
+
+
+footer {
+  float:left !important;
+}
+
+p {
+  font-weight:300 !important;
+}
+
+.mobile {
+
+  display:none;
+
+}
+
+.desktop {
+  display:block;
+
+}
+
+body {
+  line-height:1.5em;
+  font-family:'roboto';
+  margin-top:1%;
+  font-weight:300;
+  letter-spacing:0.03em;
+  font-size:15px;
+  margin:0 auto;
+
+}
+
+.banner1 {
+position:relative;
+float:left;
+
```

```suggestion
width: 100%;
```

Maybe it will be good to add a width of 100% to the `.banner1` as well. Otherwise the image can't be full width.